### PR TITLE
Fix pylint pragmas to conform to new rules.

### PR DESCRIPTION
**Pull request by robrap**

### CODE REVIEW

**Comment by robrap** [comment]
@jmbowman Is there a way to suppress failures for unnecessary pragmas and should we do that?

File: {{cookiecutter.repo_name}}/setup.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# pylint: disable=C0111,W6005,W6100
+# pylint: disable=C0111
```
Comment (jmbowman) [inline {{cookiecutter.repo_name}}/setup.py]: Huh, do we really not need the W6100 suppression anymore?  That's the one from `caniusepython3` that complains about string literals without `b` or `u` prefixes when not using `from __future__ import unicode_literals`.  And `setup.py` doesn't work well in Python 2 with unicode literals or Python 3 with byte literals.

**Comment by robrap** [comment]
@jmbowman This is different from what I saw locally.  This makes it seem like I removed too much, but locally it was complaining that I had these defined. :(

**Comment by robrap** [comment]
@jmbowman: Is this something you already took care of?  Should I close this PR?

**Comment by jmbowman** [comment]
I don't think I made any specific changes around this; it's just that `useless-suppression` isn't considered a problem worth failing the pylint run, so the tests pass.  It's probably still showing up in the text output under Python 3, which is a nuisance that we ought to fix.

**Comment by robrap** [comment]
Welcome back @jmbowman.

I don't plan on doing anything with this PR at this point.  I can close it and leave the branch for you, or close it and delete the branch for now, or leave it open if you plan on taking this over.  Just let me know.  Thanks.

**Comment by robrap** [comment]
Closing this pull request.
FYI: @jmbowman

**Pull request closed without merging**